handbook/marketing/README.md

```diff
@@ -40,38 +40,22 @@ Here are some topics for social media posts:
 - Industry news about device management
 - Upcoming events, interviews, and podcasts
 
-
-
-
 ## Responsibilities
 
 ### Process pending swag requests from the website
-TODO
-
-### Send one-off swag
-TODO
-
-<!--
-### Swag requests
-We deliver Fleet swag for those who request it through [Fleet documentation](https://fleetdm.com/docs) and may occasionally have upstanding community members and customers who are in need of some cool Fleet swag as well.
-
-#### Fulfilling Printful orders
 On a weekly basis, Marketing must log in to [Printful](https://www.printful.com/) and complete any draft orders so that they are sent. We currently only ship within the 48 contiguous United States and approve one free order per household under normal circumstances.
-
 To do this:
-
 1. Go to the ["Orders"](https://www.printful.com/ca/dashboard/default/orders) page in Printful
 2. Look for an order status of "Draft" and click that order.
 3. Copy the address displayed in the draft order. Close the modal.
 4. Under "Order list", there's a search box for customers that you will need to paste the address.
 5. The results should only display one order. Any additional could be duplicate orders or those made in error.
 6. Repeat this process until all remaining draft orders show "Waiting for Fulfillment"
 
-#### Fulfilling Postal orders
-
+### Send one-off swag
 There are many times in which community members, customers, and contributors are in need of some cool Fleet swag. We use [Postal](https://www.postal.com/) to fulfill these swag requests. Postal warehouses Fleet swag that is available to send directly. There are two main ways to send swag through Printful. Direct sends and Magic Links.
 
-##### Magic Link
+**Magic Link**
```

Comment:
```suggestion
**Magic links**: This is a great
```

this is because they are rendered on one line anyway, so this way it's not confusing when you're editing the markdown.